integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/transformMediatorNamespaceProperties.xml

```diff
@@ -0,0 +1,14 @@
+<proxy xmlns="http://ws.apache.org/ns/synapse" name="transformMediatorNamespaceProperties" startOnLoad="true" transports="https http">
+<description/>
```

Comment:
Shall we add licence headers for all the applicable files

Reply:
Done

Reply:
done